### ISSUE TITLE
Minor Patient Record UI fixes

### ISSUE THREAD

**Issue by sksmi** [SACGF/variantgrid#221]
**Background**
People are only rarely using the patient pages and/or using them incorrectly, working on trying to encourage best practice, however, some of the fields and behaviour are non-intuitive for research. Most likely occurs because fields optimised for clinical integration. Suggestions below are for quick (easy?), minor fixes with reasonable user impact. Discussion around optimising the patient-specimen pages for some of the previously discussed patient registry work will be logged as a separate issue at a future date.

**Suggested fixes**

Create new patient: 
- When manually creating a new patient, after clicking create patient button page new form is displayed. Would be great to instead go to the info page for the patient that has just been created as often want to add sample & specimen info & links. Annoying to have to find the patient in the grid. 
- 'Code' can't be entered by users and is blank on the patient grid. Can only see family ID, which isn't that helpful if you've got a big family. Would be great to show patient ID in research.
![image.png](https://images.zenhubusercontent.com/5bc5465086f3a678d497c0be/7dca6619-553f-4aab-8af4-c356f114a040)
- Reset button needs some love (text, box position) 
- Would be good to have a deceased y/n option, as often know patient is deceased, but don't know DOD. Also to match patient csv upload.
- Still confused about patient name or code field - often would like to add both, separate concepts. Definitely would like patient code on classification form for both research & clinical. One for later discussion?

View patient/patient tab:
- 2x phenotypes headers shown, maybe consanguineous & affected could go above population?
![image.png](https://images.zenhubusercontent.com/5bc5465086f3a678d497c0be/2eb77e7e-2ab5-4bb4-ae27-869f6e3ab134)![image.png](https://images.zenhubusercontent.com/5bc5465086f3a678d497c0be/358581cb-8883-42f2-88bb-d8cacbe72cd7)

View patient/specimen tab:
- not clear how to add multiple specimens as once details for first specimen have been added, the 'patient specimen saved successfully' message appears, but page needs to be refreshed to add a new specimen - not sure many people get to the point of working this out. 
- would be good to have some sort of formatting separating info from different specimens as not clear at first glance how many specimens you're looking at or where the specimen changes - header? horizontal line? box? 
![image.png](https://images.zenhubusercontent.com/5bc5465086f3a678d497c0be/53ff21d0-6e4b-4315-913b-3762c04e374d)

**Additional context**
Medium priority  - no one is really using the patient sections, but think this should be encouraged as missing advantages of HPO/OMIM integration.

**Comment by davmlaw** [comment]
Yes this was deliberately crippled for privacy / to avoid incidental findings in diagnostics Code is an "externally managed ID" - ie set by something like HELIX for SA Path.

We should allow some config for research to be able to modify this

**Comment by sksmi** [comment]
Would it be solved if we have 2 patient IDs? 
code = externally managed ID  (ie helix etc)
patient ID = a manually managed ID (ie. study ID, GF, MRN, UR...  whatever is needed by the project)

**Comment by davmlaw** [comment]
Moved comment to #230

**Comment by sksmi** [comment]
More minor suggestions View patient/patient tab:

- Can we drop the ontology prefix on the tooltip?
- Not sure how we're supposed to be using the gene dropdown now - I don't think anyone will know the gene ID. Searching using the gene symbol works, but gives you the gene ID which isn't matched. Also seems to give me only 1 gene ID (for gene symbol: C5) regardless of what I choose. 

![image.png](https://images.zenhubusercontent.com/5bc5465086f3a678d497c0be/323b6f7e-8215-48a2-a2f2-1e4306a2c9ec)

![image.png](https://images.zenhubusercontent.com/5bc5465086f3a678d497c0be/add7229c-5fa0-45be-9273-17ed71d83231)

**Comment by davmlaw** [comment]
Copied issue about gene dropdown and ontology prefix to #243

**Comment by sksmi** [comment]
https://variantgrid.com/patients/view_patient/1516
Can't make the clinical notes box wider, issue when there's lots of text:
(also an issue on the new patient form)
![image.png](https://images.zenhubusercontent.com/5bc5465086f3a678d497c0be/47e4f87e-deb4-4879-9611-a26847b83521)
